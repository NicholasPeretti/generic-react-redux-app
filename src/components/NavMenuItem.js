import React from 'react'
import Style from '../style'
import styled from 'styled-components'

const StyledLi = styled.li`
  float: right;
  padding: ${Style.spacing.padding2};
`

var NavMenuItem = ({ children }) => (
  <StyledLi>
    {children}
  </StyledLi>
)

export default NavMenuItem
