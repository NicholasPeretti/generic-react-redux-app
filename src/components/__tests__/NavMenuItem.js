import React from 'react'
import { shallow } from 'enzyme'
import PresentationalTest from '../../utils/tests/presentational'
import NavMenuItem from '../NavMenuItem'

describe('NavMenuItem', () => {
  PresentationalTest(NavMenuItem)

  test('Should match snapshot', () => {
    const wrapper = shallow(<NavMenuItem/>)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
