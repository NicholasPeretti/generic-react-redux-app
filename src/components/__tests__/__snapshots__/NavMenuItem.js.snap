// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavMenuItem Should match snapshot 1`] = `"<li class=\\"sc-bdVaJa eqdneR\\"></li>"`;
